integration-tests/src/test/resources/features/end2endtestit.feature

```diff
@@ -107,3 +107,75 @@ Feature: End to End Bridge integration tests
       ]
     }
     """
+
+  Scenario: Slack Action Processor is created and slack message should be received in the slack channel
+    Given authenticate against Manager
+
+    When create a new Bridge "mybridge"
+    And the Bridge "mybridge" is existing with status "ready" within 2 minutes
+    And the Ingress of Bridge "mybridge" is available within 2 minutes
+
+    Then add a Processor to the Bridge "myBridge" with body:
+    """
+    {
+      "name": "slackProcessor",
+      "action": {
+        "type": "Slack",
+        "parameters": {
+            "channel": "mc",
+            "webhookUrl": "https://hooks.slack.com/services/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"
+            }
+      },
+      "filters": [
+        {
+        "key": "source",
+        "type": "StringEquals",
+        "value": "StorageService"
+        }
+       ],
+  	  "transformationTemplate": "{data.myMessage}"
+    }
+    """
+    And the Processor "slackProcessor" of the Bridge "mybridge" is existing with status "ready" within 3 minutes
+    And the Processor "slackProcessor" of the Bridge "mybridge" has action of type "Slack"
+
+    Then add a Processor to the Bridge "mybridge" and returns HTTP response code 400 with body:
+    """
+    {
+      "name": "processorInvalid",
+      "action": {
+        "type": "SlackAction",
+        "properties": {
+            "channel": "mc",
+            "webhookUrl": "https://hooks.slack.com/services/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"
```

Comment:
We probably shouldn't put this webhook URL in the GH repo

Reply:
Sure will place in GitHub secret